scripts/linux/electron-installer-appimage.sh

```diff
@@ -105,7 +105,6 @@ fi
 if [ -z "$TMPDIR" ]; then
   TMPDIR=$(mktemp -d)
```

Comment:
The intention is to re-use `$TMPDIR` from the environment if its there, otherwise use `mktemp -d`. We are manually passing a custom `TMPDIR` from the Makefile, but this script might get ran in a completely standalone fashion, in which case we would have to document that this script depends on a certain environment setting.